components/sslh/Makefile

```diff
@@ -0,0 +1,65 @@
+#
+# CDDL HEADER START
+#
+# The contents of this file are subject to the terms of the
+# Common Development and Distribution License (the "License").
+# You may not use this file except in compliance with the License.
+#
+# You can obtain a copy of the license at usr/src/OPENSOLARIS.LICENSE
+# or http://www.opensolaris.org/os/licensing.
+# See the License for the specific language governing permissions
+# and limitations under the License.
+#
+# When distributing Covered Code, include this CDDL HEADER in each
+# file and include the License file at usr/src/OPENSOLARIS.LICENSE.
+# If applicable, add the following below this CDDL HEADER, with the
+# fields enclosed by brackets "[]" replaced with your own identifying
+# information: Portions Copyright [yyyy] [name of copyright owner]
+#
+# CDDL HEADER END
+
+#
+# Copyright (c) 2011, 2012, Oracle and/or its affiliates. All rights reserved.
+#
+
+include ../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=		sslh
+COMPONENT_VERSION=	1.17
+COMPONENT_PROJECT_URL=	http://www.rutschle.net/tech
+COMPONENT_SRC=		$(COMPONENT_NAME)-v$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_HASH=	\
+    sha256:4f3589ed36d8a21581268d53055240eee5e5adf02894a2ca7a6c9022f24b582a
+COMPONENT_ARCHIVE_URL=	http://www.rutschle.net/tech/$(COMPONENT_ARCHIVE)
+COMPONENT_BUGDB=	utility/sslh
+
+LD_OPTIONS=-lresolv -lsocket -lnsl
+
+include ../../make-rules/prep.mk
+include ../../make-rules/justmake.mk
+include ../../make-rules/ips.mk
+
+# remove the unwanted file.
+COMPONENT_PREP_ACTION = rm -f $(SOURCE_DIR)/match.S
+
+COMPONENT_BUILD_ARGS += -f Makefile
+COMPONENT_BUILD_TARGETS += CC=gcc
```

Comment:
I don't think BUILD_TARGETS is a good way to set CC. Why also CC=gcc and not $(CC)? I'd set it in COMPONENT_BUILD_ENV